aiida/backends/djsite/db/subtests/nodes.py

```diff
@@ -96,7 +96,8 @@ def test_links_and_queries(self):
 class TestTransitiveClosureDeletionDjango(AiidaTestCase):
```

Comment:
These tests should now be backend-independent, right? Can we move them to the set of backend-independent tests?

Reply:
They should be, but there's no way to delete links backend independently. This kind of test relies on creating links and deleting them again, and this is done via the backend-specific models.